src/components/LogInForm/LogInForm.scss

```diff
@@ -1,19 +1,38 @@
-section {
+.login {
     margin: auto;
     width: 80%; 
     padding-top: 1em;
     -webkit-box-shadow: 0px 11px 34px -10px rgba(74,74,74,0.42);
     -moz-box-shadow: 0px 11px 34px -10px rgba(74,74,74,0.42);
     box-shadow: 0px 11px 34px -10px rgba(74,74,74,0.42);
+    text-align: center;
 }
 form {
     width: 100%;
     margin: auto;
 }
 button {
```

Comment:
Could you change the `button`, `form`, `button:hover` selectors to be nested inside of  `.login{ ... }` ? Otherwise there will be overridden styles once again :astonished: